staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation.go

```diff
@@ -40,6 +41,14 @@ const (
 	// OldScopedVarName is the variable name assigned to the existing value of the locally scoped data element of a
 	// CEL validation expression.
 	OldScopedVarName = "oldSelf"
+
+	// perCallLimit specify the actual cost limit per CEL validation call
+	//TODO: pick the number for perCallLimit
+	perCallLimit = uint64(math.MaxInt64)
+
+	// RuntimeCELCostBudget is the overall cost budget for runtime CEL validation cost per CustomResource
+	//TODO: pick the RuntimeCELCostBudget
+	RuntimeCELCostBudget = math.MaxInt64
```

Comment:
Thanks for pointing out. I am not quite sure if CustomResource validation related field is also part of API review([ref doc](https://github.com/kubernetes/community/blob/master/sig-architecture/api-review-process.md#what-parts-of-a-pr-are-api-changes))?

Reply:
It's part of our API surface, falls under "validation (pkg/apis/*/validation.go or OpenAPI for CRDs)"

Reply:
Thanks. I have moved it under `k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation` to trigger api review. Please let me know if other place is preferred^-^

Reply:
to minimize churn, I'm happy for a no-op move to happen in a follow-up... this seemed like it was close otherwise

Reply:
Thank you. I will do it in the follow up PR since I still need to sort out some dependency issue in terms of where to put it. It would be great if we could get this one merged first :)